bytecomp/matching.ml

```diff
@@ -163,7 +163,7 @@ end = struct
 
   let empty = []
 
-  let start n : t = [ { left = []; right = omegas n } ]
+ let start n : t = [ { left = []; right = omegas n } ]
```

Comment:
This shouldn't be in the diff, fatfingers on my part, but it will disappear before being sent upstream as I'll be calling ocamlformat.